README.md

```diff
@@ -1,6 +1,6 @@
 # autopredictor
 
-A package that streamline the repetitive task of regression models selection and comparison in the machine learning workflow.
+The Autopredictor Python package streamlines the process of selecting and assessing machine learning models, presenting a simplified approach for evaluating different regression models without intricate manual setup. This tool facilitates the exploration of multiple models on a dataset, minimizing the coding required for selecting and fitting various models. Utilizing preprocessed and trained data, the package evaluates models using default settings, enabling users to swiftly comprehend model performance. By computing and showcasing diverse performance metrics for each model, it offers an efficient means to compare their effectiveness. Overall, Autopredictor provides a convenient and quick framework for initial model assessment and comparison in machine learning workflows.
```

Comment:
Very well written! Do you think we should say 'This Autotpredictor package' and 'this package evaluates ...' instead of 'the package' so it is consistent with 'This tool' and the rest?

Reply:
I have made changes as per suggestion.